flask/datahandle.py

```diff
@@ -5,6 +5,32 @@
 
 ###Helper functions
 
+
+def formatItem(item):
```

Comment:
I don't think you need this method, you can just use [rstrip](https://www.w3schools.com/python/ref_string_rstrip.asp).

txt.rstrip(' ,')

Reply:
I made the function primarily for the larger string ("Food item 1, food item 2, food item 3") and I found it more reliable to make a custom function for when there is an extra comma and/or space(s) because of how the MDining API formats certain values. rstrip was fine for individual items.

Reply:
I'm actually going to use the rfind you mentioned below to alleviate this alongside rstrip, makes it work.